sources/designspace/Besley.designspace

```diff
@@ -251,14 +251,18 @@
 
               <dict>
                 <key>name</key>
-                <string>Regular</string>
```

Comment:
@EwonRael I need to make sure you're OK with these changes. These are the names Google insists on.